### PR TITLE
add aks rotate-certs command

### PR DESCRIPTION
**Pull request by andyzhangx**
add aks rotate-certs command

---

This checklist is used to make sure that common guidelines for a pull request are followed.

### General Guidelines

- [x] Have you run `azdev style <YOUR_EXT>` locally? (`pip install azdev` required)
- [x] Have you run `python scripts/ci/test_index.py -q` locally?

For new extensions:

- [x] My extension description/summary conforms to the [Extension Summary Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/extensions/extension_summary_guidelines.md).

### CODE REVIEW

**Comment by andyzhangx** [comment]
It looks like python sdk does not contain `rotate_cluster_certificates`, how can I update to latest aks sdk @zqingqing1 @weinong 
```
# az aks rotate-certs -g $RESOURCE_GROUP_NAME -n $CLUSTER_NAME
Kubernetes will be unavailable during certificate rotation process.
Are you sure you want to perform this operation? (y/n): y
The command failed with an unexpected error. Here is the traceback:

'ManagedClustersOperations' object has no attribute 'rotate_cluster_certificates'
Traceback (most recent call last):
  File "/opt/az/lib/python3.6/site-packages/knack/cli.py", line 206, in invoke
    cmd_result = self.invocation.execute(args)
  File "/opt/az/lib/python3.6/site-packages/azure/cli/core/commands/__init__.py", line 560, in execute
    raise ex
  File "/opt/az/lib/python3.6/site-packages/azure/cli/core/commands/__init__.py", line 618, in _run_jobs_serially
    results.append(self._run_job(expanded_arg, cmd_copy))
  File "/opt/az/lib/python3.6/site-packages/azure/cli/core/commands/__init__.py", line 611, in _run_job
    six.reraise(*sys.exc_info())
  File "/opt/az/lib/python3.6/site-packages/six.py", line 693, in reraise
    raise value
  File "/opt/az/lib/python3.6/site-packages/azure/cli/core/commands/__init__.py", line 588, in _run_job
    result = cmd_copy(params)
  File "/opt/az/lib/python3.6/site-packages/azure/cli/core/commands/__init__.py", line 297, in __call__
    return self.handler(*args, **kwargs)
  File "/opt/az/lib/python3.6/site-packages/azure/cli/core/__init__.py", line 461, in default_command_handler
    return op(**command_args)
  File "/root/.azure/cliextensions/aks-preview/azext_aks_preview/custom.py", line 1917, in aks_rotate_certs
    return sdk_no_wait(no_wait, client.rotate_cluster_certificates, resource_group_name, name)
AttributeError: 'ManagedClustersOperations' object has no attribute 'rotate_cluster_certificates'
```

**Comment by andyzhangx** [comment]
I see, just copy `azure-sdk-for-python/./sdk/containerservice/azure-mgmt-containerservice/azure/mgmt/containerservice/v2019_10_01/operations/*.py` to this repo.

**Comment by andyzhangx** [comment]
it works now, ready for review  
cc @sauryadas

**Comment by andyzhangx** [comment]
failed on this, looks like not related to my PR:
```
Finished dev setup.
+pip install -e 'git+https://github.com/Azure/azure-cli@dev#egg=azure-cli-dev-tools&subdirectory=tools' -q
+echo Installed.
Installed.
+az --version
Traceback (most recent call last):
  File "/home/travis/virtualenv/python3.6.3/lib/python3.6/site-packages/pkg_resources/__init__.py", line 573, in _build_master
    ws.require(__requires__)
  File "/home/travis/virtualenv/python3.6.3/lib/python3.6/site-packages/pkg_resources/__init__.py", line 891, in require
    needed = self.resolve(parse_requirements(requirements))
  File "/home/travis/virtualenv/python3.6.3/lib/python3.6/site-packages/pkg_resources/__init__.py", line 782, in resolve
    raise VersionConflict(dist, req).with_context(dependent_req)
pkg_resources.ContextualVersionConflict: (azure-mgmt-nspkg 1.0.0 (/home/travis/virtualenv/python3.6.3/lib/python3.6/site-packages), Requirement.parse('azure-mgmt-nspkg>=2.0.0'), {'azure-mgmt-keyvault', 'azure-mgmt-iothubprovisioningservices', 'azure-mgmt-devtestlabs', 'azure-mgmt-consumption', 'azure-mgmt-maps', 'azure-mgmt-loganalytics', 'azure-mgmt-billing', 'azure-mgmt-batchai', 'azure-mgmt-applicationinsights', 'azure-mgmt-dns', 'azure-mgmt-datamigration', 'azure-mgmt-relay', 'azure-mgmt-msi', 'azure-mgmt-servicefabric', 'azure-mgmt-monitor'})
During handling of the above exception, another exception occurred:
Traceback (most recent call last):
  File "/home/travis/virtualenv/python3.6.3/bin/az", line 4, in <module>
    __import__('pkg_resources').require('azure-cli==2.0.76')
  File "/home/travis/virtualenv/python3.6.3/lib/python3.6/site-packages/pkg_resources/__init__.py", line 3098, in <module>
    @_call_aside
  File "/home/travis/virtualenv/python3.6.3/lib/python3.6/site-packages/pkg_resources/__init__.py", line 3082, in _call_aside
    f(*args, **kwargs)
  File "/home/travis/virtualenv/python3.6.3/lib/python3.6/site-packages/pkg_resources/__init__.py", line 3111, in _initialize_master_working_set
    working_set = WorkingSet._build_master()
  File "/home/travis/virtualenv/python3.6.3/lib/python3.6/site-packages/pkg_resources/__init__.py", line 575, in _build_master
    return cls._build_from_requirements(__requires__)
  File "/home/travis/virtualenv/python3.6.3/lib/python3.6/site-packages/pkg_resources/__init__.py", line 588, in _build_from_requirements
    dists = ws.resolve(reqs, Environment())
  File "/home/travis/virtualenv/python3.6.3/lib/python3.6/site-packages/pkg_resources/__init__.py", line 777, in resolve
    raise DistributionNotFound(req, requirers)
pkg_resources.DistributionNotFound: The 'azure-mgmt-nspkg>=2.0.0' distribution was not found and is required by azure-mgmt-keyvault, azure-mgmt-iothubprovisioningservices, azure-mgmt-devtestlabs, azure-mgmt-consumption, azure-mgmt-maps, azure-mgmt-loganalytics, azure-mgmt-billing, azure-mgmt-batchai, azure-mgmt-applicationinsights, azure-mgmt-dns, azure-mgmt-datamigration, azure-mgmt-relay, azure-mgmt-msi, azure-mgmt-servicefabric, azure-mgmt-monitor
The command "./scripts/ci/verify_modified_index.sh" exited with 1.
```

File: src/aks-preview/HISTORY.md [hunk 1]
```diff
@@ -2,6 +2,10 @@
 
 Release History
 ===============
+0.4.21
++++++
+* Support cluster certificate rotation operation using "az aks rotate-certs"
```
Comment (xizhamsft) [inline src/aks-preview/HISTORY.md]: Could you also add this line : Add support for `az aks kanalyze`

Comment (andyzhangx) [reply src/aks-preview/HISTORY.md]: done

**Comment by andyzhangx** [comment]
hi @zikalino, could you take a review? Thanks. and about the test failure, looks like it's not related to my PR, thanks.

```
```
Finished dev setup.
+pip install -e 'git+https://github.com/Azure/azure-cli@dev#egg=azure-cli-dev-tools&subdirectory=tools' -q
+echo Installed.
Installed.
+az --version
Traceback (most recent call last):
  File "/home/travis/virtualenv/python3.6.3/lib/python3.6/site-packages/pkg_resources/__init__.py", line 573, in _build_master
    ws.require(__requires__)
  File "/home/travis/virtualenv/python3.6.3/lib/python3.6/site-packages/pkg_resources/__init__.py", line 891, in require
    needed = self.resolve(parse_requirements(requirements))
  File "/home/travis/virtualenv/python3.6.3/lib/python3.6/site-packages/pkg_resources/__init__.py", line 782, in resolve
    raise VersionConflict(dist, req).with_context(dependent_req)
pkg_resources.ContextualVersionConflict: (azure-mgmt-nspkg 1.0.0 (/home/travis/virtualenv/python3.6.3/lib/python3.6/site-packages), Requirement.parse('azure-mgmt-nspkg>=2.0.0'), {'azure-mgmt-keyvault', 'azure-mgmt-iothubprovisioningservices', 'azure-mgmt-devtestlabs', 'azure-mgmt-consumption', 'azure-mgmt-maps', 'azure-mgmt-loganalytics', 'azure-mgmt-billing', 'azure-mgmt-batchai', 'azure-mgmt-applicationinsights', 'azure-mgmt-dns', 'azure-mgmt-datamigration', 'azure-mgmt-relay', 'azure-mgmt-msi', 'azure-mgmt-servicefabric', 'azure-mgmt-monitor'})
During handling of the above exception, another exception occurred:
Traceback (most recent call last):
  File "/home/travis/virtualenv/python3.6.3/bin/az", line 4, in <module>
    __import__('pkg_resources').require('azure-cli==2.0.76')
  File "/home/travis/virtualenv/python3.6.3/lib/python3.6/site-packages/pkg_resources/__init__.py", line 3098, in <module>
    @_call_aside
  File "/home/travis/virtualenv/python3.6.3/lib/python3.6/site-packages/pkg_resources/__init__.py", line 3082, in _call_aside
    f(*args, **kwargs)
  File "/home/travis/virtualenv/python3.6.3/lib/python3.6/site-packages/pkg_resources/__init__.py", line 3111, in _initialize_master_working_set
    working_set = WorkingSet._build_master()
  File "/home/travis/virtualenv/python3.6.3/lib/python3.6/site-packages/pkg_resources/__init__.py", line 575, in _build_master
    return cls._build_from_requirements(__requires__)
  File "/home/travis/virtualenv/python3.6.3/lib/python3.6/site-packages/pkg_resources/__init__.py", line 588, in _build_from_requirements
    dists = ws.resolve(reqs, Environment())
  File "/home/travis/virtualenv/python3.6.3/lib/python3.6/site-packages/pkg_resources/__init__.py", line 777, in resolve
    raise DistributionNotFound(req, requirers)
pkg_resources.DistributionNotFound: The 'azure-mgmt-nspkg>=2.0.0' distribution was not found and is required by azure-mgmt-keyvault, azure-mgmt-iothubprovisioningservices, azure-mgmt-devtestlabs, azure-mgmt-consumption, azure-mgmt-maps, azure-mgmt-loganalytics, azure-mgmt-billing, azure-mgmt-batchai, azure-mgmt-applicationinsights, azure-mgmt-dns, azure-mgmt-datamigration, azure-mgmt-relay, azure-mgmt-msi, azure-mgmt-servicefabric, azure-mgmt-monitor
The command "./scripts/ci/verify_modified_index.sh" exited with 1.
```
```

**Comment by sauryadas** [comment]
@qwordy Can you please help merge this? We really need to roll this out as aks certificates start expring next week

**Comment by qwordy** [comment]
> 
> 
> @qwordy Can you please help merge this? We really need to roll this out as aks certificates start expring next week

OK. But Some checks were not successful. And we need an approval by reviewer with write access.

**Comment by andyzhangx** [comment]
> > @qwordy Can you please help merge this? We really need to roll this out as aks certificates start expring next week
> 
> OK. But Some checks were not successful. And we need an approval by reviewer with write access.

@qwordy the test failure is not related to this PR, you could see latest 4 PRs all failed with same failure.
@zikalino could you take a review? Thanks.

**Comment by sauryadas** [comment]
@zikalino many thanks!!

**Pull request merged**